airbyte_cdk/sources/streams/concurrent/default_stream.py

```diff
@@ -23,7 +19,6 @@ def __init__(
         partition_generator: PartitionGenerator,
         name: str,
         json_schema: Mapping[str, Any],
-        availability_strategy: AbstractAvailabilityStrategy,
```

Comment:
This is in theory a breaking change but the only other usage I could find of DefaultStream is in source-salesforce but it uses `StreamFacade.create_from_stream` so we should be good